include/MacportsLegacySupport.h

```diff
@@ -186,7 +186,7 @@
 /*  realpath() on < 1060 does not support modern NULL buffer usage */
 #define __MP_LEGACY_SUPPORT_REALPATH_WRAP__   (__MPLS_TARGET_OSVER < 1060)
 
-/* setattrlistat */
+/* fsetattrlistat, fgetattrlistat */
```

Comment:
There's a trivial lexical conflict here with the updated `master`, due to non-conflicting changes on adjacent lines.